Move xml doc section (#24705)
dotnet/docs: docs/csharp/misc/cs1570.md

When using [**DocumentationFile**](../language-reference/compiler-options/output.md#documentationfile), any comments in the source code must be in XML. Any error with your XML markup will generate CS1570. For example:

- If you are passing a string to a **cref**, such as in an [\<exception>](../language-reference/xmldoc/recommended-tags.md) tag, the string must be enclosed in double quotation marks.

> - If you are using a tag, such as [\<seealso>](../language-reference/xmldoc/recommended-tags.md), which does not have a closing tag, you must specify a forward slash before the closing angle bracket.

Reviewer: ```suggestion
- If you're using a tag that doesn't have a closing tag, such as [\<seealso>](../language-reference/xmldoc/recommended-tags.md), you must specify a forward slash before the closing angle bracket.
```
